hw_1.py

```diff
@@ -0,0 +1,21 @@
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
```

Comment:
There is not a single comment in this file. This makes it very difficult to review